src/Engine/GameInput/SelectCargo.java

```diff
@@ -77,7 +77,7 @@ protected OptionSet initOptions()
   @Override
   public void back()
   {
-    if( myStateData.unitLocationMap.isEmpty() )
+    if( null != myStateData.unitLocationMap && myStateData.unitLocationMap.isEmpty() )
     {
       myStateData.unitLocationMap = null;
```

Comment:
I'm... curious what this change is for?

Reply:
`unitLocationMap` can be null if you back out of an unload action after entering the location-selection state for the first unit.

Reply:
...Oh. I missed that the other condition was *accessing* it, for some reason.